templates/definition/tariff/forecast-solar.yaml

```diff
@@ -0,0 +1,31 @@
+template: forecast-solar
+products:
+  - brand: Forecast.Solar
+requirements:
+  evcc: ["skiptest"]
+group: solar
+params:
+  - name: lat
```

Comment:
Sollten lat und lon auch in die Liste der `redactSecrets`?

Reply:
As you like...